dot-net-sdk/store/IConfigurationStore.cs

```diff
@@ -5,9 +5,9 @@ namespace eppo_sdk.store;
 
 public interface IConfigurationStore
 {
-    void FetchConfiguration();
-    Flag? GetExperimentConfiguration(string key);
+    void LoadConfiguration();
```

Comment:
Totally being overly sematic here - `Fetch` implies something is returned to the caller.